### DISCUSSION TITLE
弄个开发交流群？

### DISCUSSION THREAD

**Discussion by RayWangQvQ** [RayWangQvQ/BiliBiliToolPro#439] [Ideas]
最近发现收到很多高质量的PR，想想都还没有针对开发来交流的地方，要不搞一个“开发群”？

区别于之前的“聊天吹水群”，可以一起聊聊bilibilitool的新想法、dotnet、其他好玩的工具、docker或者云技术等等（这样看好像还是在吹水。。。）

**Comment by Sibadaer** [comment]
同意 如果国内环境不安全可以搞个tg交流群

**Comment by zheshi111** [comment]
弄个tg交流群

**Comment by RayWangQvQ** [comment]
tg群一直都有，相比微信群冷清很多，可能平时都还是微信用的多

**Reply by Sibadaer** [reply]
微信群是那个， 可以拉我进群吗？

**Reply by RayWangQvQ** [reply]
楼下

**Comment by RayWangQvQ** [comment]
#469

**Reply by elricccc** [reply]
可以再发下吗，微信群二维码过期了
